overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake

```diff
@@ -22,7 +22,7 @@ if("${_target_type}" STREQUAL "STATIC_LIBRARY")
     find_package(harfbuzz CONFIG)
 
     set_property(TARGET Qt5::Core APPEND PROPERTY INTERFACE_LINK_LIBRARIES
-        ZLIB::ZLIB JPEG::JPEG PNG::PNG Freetype::Freetype SQLite3::SQLite3 harfbuzz::harfbuzz
+        ZLIB::ZLIB JPEG::JPEG PNG::PNG Freetype::Freetype SQLite::SQLite3 harfbuzz::harfbuzz
```

Comment:
Are you sure this is correct?

In Mixxx `CMakeLists.txt`, we also link to `SQLite3::SQLite3`...

Reply:
This is changed in https://github.com/mixxxdj/mixxx/pull/4432

Reply:
ok